taskfiles/deps.yaml

```diff
@@ -0,0 +1,151 @@
+version: "3"
+
+tasks:
+  install-all:
+    desc: "Install all dependencies required by clp-ffi-js."
+    run: "once"
+    cmds:
+      - "rm -rf '{{.G_DEPS_CMAKE_SETTINGS_DIR}}'"
+      - "mkdir -p '{{.G_DEPS_CMAKE_SETTINGS_DIR}}'"
+      - task: ":utils:cmake:install-deps-and-generate-settings"
+        vars:
+          CMAKE_SETTINGS_DIR: "{{.G_DEPS_CMAKE_SETTINGS_DIR}}"
+          CMAKE_SETTINGS_FILE: "{{.G_DEPS_CMAKE_SETTINGS_FILE}}"
+          DEP_TASK: "deps:install-all-parallel"
+
+  install-all-parallel:
+    internal: true
+    deps:
+      - "download-boost"
+      - "download-clp"
+      - "download-fmt"
+      - "download-nlohmann-json"
+      - "download-spdlog"
+      - "download-ystdlib"
+      - "download-zstd"
+
+  download-boost:
+    internal: true
+    vars:
+      LIB_NAME: "boost"
+      BOOST_OUTPUT_DIR: "{{.G_DEPS_DIR}}/{{.LIB_NAME}}-src"
+    run: "once"
+    cmds:
+      - task: ":utils:remote:download-and-extract-tar"
+        vars:
+          FILE_SHA256: "d6c69e4459eb5d6ec208250291221e7ff4a2affde9af6e49c9303b89c687461f"

```

Comment:
for some reason the SHA sums are not validated when the tasks run (i didn't see any error).

command i used to generate the sums:
```
curl -sL <URL> | sha256sum
```